bazel-java-deps.bzl

```diff
@@ -104,8 +104,8 @@ def install_java_deps():
             "org.junit.jupiter:junit-jupiter-engine:5.0.0",
             "org.junit.platform:junit-platform-engine:1.0.0",
             "org.junit.platform:junit-platform-runner:1.0.0",
```

Comment:
@miklos-da think you have to rerun bazel run @unpinned_maven//:pin and check in the gen'd maven_install.json file from that command

Reply:
Thanks for the note -- will do that.